kube-fluentd/README.md

```diff
@@ -0,0 +1,16 @@
+# Overview
+
+This directory contains docker images with fluentd, which can be later configured to collect
+log from any source.
+
+# Usage
+
+Each image is built with its own set of plugins which you can later use in configuration. Set of
+plugin is enumerated in Gemfile. You can find details about fluentd configuration on the
+[official site](http://docs.fluentd.org/articles/config-file).
+
+In order to configure fluentd image, you should mount directory with `.conf` files to
+`/etc/fluent/config` or add files to this directory by building a new image on top. All `.conf`
```

Comment:
Traditionally, directories with extra configuration files have the `.d` suffix, so how about `/etc/fluentd/config.d`?